<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrarPM</class>
 <widget class="QDialog" name="RegistrarPM">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro Nuevo Usuario</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="Registrar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>481</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item row="0" column="0" rowspan="4">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="lNombre">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>19</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lApellido">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>19</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Apellido:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lCorreo">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>19</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Correo:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lPosicion">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>19</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Posici√≥n:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineApellido">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Carlos</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineCorreo">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Carlos</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="Cposicion">
      <item>
       <property name="text">
        <string>Project Manager</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Comercial Project Manager</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineNombre">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Carlos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lTitulo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>300</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registra Nuevo PM o CPM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
